### PR TITLE
feat: Basic metrics for minidump scrubbing

### PR DESCRIPTION
**Pull request by flub**
A dump counter keeping track of minidump scrubbing.

I'm considering this as part of the larger minidump scrubbing work, so:

#skip-changelog

### CODE REVIEW

**Review by untitaker — APPROVED** [review]

File: relay-server/src/actors/events.rs [hunk 1]
```diff
@@ -1019,8 +1019,17 @@ impl EventProcessor {
...
+                        counter(RelayCounters::MinidumpsScrubbed) += 1,
+                        status = "error"
+                    );
+                    log::warn!("failed to scrub minidump: {}", LogError(&scrub_error));
                     processor.scrub_attachment(filename, &mut payload);
+                } else {
+                    metric!(
+                        counter(RelayCounters::MinidumpsScrubbed) += 1,
+                        status = "ok"
+                    );
```
Comment (jan-auer) [inline relay-server/src/actors/events.rs]: This will also increment even if no rules were applied. Should we track this?

Comment (flub) [reply relay-server/src/actors/events.rs]: Good point, I've improved this now.

File: relay-server/src/actors/events.rs [hunk 2]
```diff
@@ -1018,9 +1018,22 @@ impl EventProcessor {
...
+                            status = if modified { "ok" } else { "n/a" },
+                        );
+                    }
+                    Err(scrub_error) => {
+                        metric!(
+                            timer(RelayTimers::MinidumpScrubbing) = start.elapsed(),
+                            status = "error"
+                        );
+                        log::warn!("failed to scrub minidump: {}", LogError(&scrub_error));
+                        processor.scrub_attachment(filename, &mut payload);
```
Comment (untitaker) [inline relay-server/src/actors/events.rs]: Can we also time this? Should be easier since this is infallible and the metric macro has block syntax

**Review by untitaker — APPROVED** [review]

File: relay-server/src/actors/events.rs (hunk 2 above)
Comment (flub) [reply relay-server/src/actors/events.rs]: I kind of wanted to time this inside `scrub_attachments` but that's in the general crate and doesn't have the metrics.  Now we'll have to remember to add this metric every time we call it :(

**Pull request merged**